cfgov/preprocessed/css/molecules/related-posts.less

```diff
@@ -0,0 +1,106 @@
+/* topdoc
+  name: Related Posts
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+    <div class="m-related-posts">
+        <h2 class="header-slug">
+            <span class="header-slug_inner">
+                Related Posts
+            </span>
+        </h2>
+        <section class="m-related-posts_section">
+            <h4>
+                <span class="cf-icon"></span>
+            </h4>
+            <ul class="list">
+                <li class="list_item">
+                    <h4 class="u-mb5">
+                        <a class="list_link" >
+                        </a>
+                    </h4>
+                    <p class="date">
+                        <span class="datetime">
+                            <time class="datetime_date" datetime="">
+                            </time>
+                        </span>
+                    </p>
+                </li>
+            </ul>
+        </section>
+    </div>
+    codenotes:
+      - |
+        Structural cheat sheet:
+        -----------------------
+        .m-related-posts
+          .header-slug
+            .header-slug_inner
+          .m-related-posts_section
+            .h4
+              .cf-icon
+            .list
+              .list-item
+                h4
+                  .list-link
+                .date
+                  .datetime
+                    .time.datetime_date
+  tags:
+    - cfgov-molecules
+*/
+
+.m-related-posts {
+
+    .list {
+
+        &.list__spaced {
+            .list_item {
+                // Optical adjustment to set total margin to 30px.
+                margin-top: unit(20px / @base-font-size-px, em);
```

Comment:
Correct, but we're no longer optically setting spacing, see https://github.com/cfpb/design-manual/issues/269